core/frontend/src/IpcApp.ts

```diff
@@ -30,15 +30,15 @@ export type PromiseReturnType<T extends AsyncFunction> = T extends (...args: any
 
 /**
  * Options for [[IpcApp.startup]]
- * @beta
+ * @public

```

Comment:
`[[IpcApp.startup]]` links to an internal API.

Reply:
I decided to make `IpcApp.startup` public with a note that it isn't intended to be called directly.

Reply:
Do you intend to push that change? Current diff shows you added `@internal` to IpcApp.startup.

Reply:
pushed